### PR TITLE
Feat: Authentication pages

### PR DESCRIPTION
**Pull request by rutu14**
### Authentication Pages
- The login page has dummy data already inserted to view the changes.
- On Login, the navbar has a avatar button.  
- On Clicking the button, dropdown appears with logout option. The navbar gets back the login button.
- On successful signup / login, screen goes to home page.
- On Errors, a alert box box appears with the error message.

[Click Here to View the Files](https://github.com/rutu14/Ecommerce-React/pull/5/commits/2a82736b615acc668714010eb6022ff86a30ce5f?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true)

### CODE REVIEW

File: src/reducers/userReducer.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const userReducer = (state, action) => {
+        switch (action.type) {
+			case "LOGIN_REQUEST":
+				return (state = {
```
Comment (santhoshpemmaka) [inline src/reducers/userReducer.js]: Suggestion:- If possible check alignment once in this file. It is useful more readable.

File: src/components/Navigation/NavLinks.jsx [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
 
-    return(
-        <div className="account">  
-            <button id="drop-btn" className="avatar avatar-text text-md cp" onClick={toggleDropdown}>RL</button>
-            {isOpen && ( <ShowDropdown/> )}
-        </div>
-    );
-}
+    const ShowDropdown = () => {
+        console.log(signout)
```
Comment (santhoshpemmaka) [inline src/components/Navigation/NavLinks.jsx]: If possible remove console statement.

**Comment by santhoshpemmaka** [comment]
Small changes need this PR. Apart from those changes remaining code looks clean and good 💯 . Good to merge 👍

**Review by santhoshpemmaka — APPROVED** [review]

File: src/reducers/userReducer.js (hunk 1 above)
Comment (rutu14) [reply src/reducers/userReducer.js]: Sure

File: src/components/Navigation/NavLinks.jsx (hunk 2 above)
Comment (rutu14) [reply src/components/Navigation/NavLinks.jsx]: Thanks for the reminder

File: src/components/Cards/CartCard.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
+const CartCard = () => {
+    return(
+<section class="cart-grid">            
+            <div class="card horizontal-large card-box-shadow cart-card m-t20">
```
Comment (dev-prabhat) [inline src/components/Cards/CartCard.jsx:4]: Try to reuse like map CartCard while passing data form Product page

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Code is fine, clean, and readable , UI looks awesome, I like the theme you have chosen

File: src/components/Cards/CartCard.jsx (hunk 3 above)
Comment (rutu14) [reply src/components/Cards/CartCard.jsx:4]: Added this page, to avoid the netlify issue. Still Working on this page.

**Pull request merged**